src/components/Header/index.tsx

```diff
@@ -103,54 +103,64 @@ function Header() {
                 }`}
               />
             </Link>
-            <div className="hidden items-center gap-x-3 text-base font-semibold text-primary lg:flex lg:text-lg">
+            <div className="hidden items-center gap-x-3 text-base font-light font-semibold text-disclaimer lg:flex lg:text-lg">
               <Link
                 href="/"
-                className={`rounded-lg border-[1.5px] border-surface px-3 py-1 transition-colors duration-300 hover:border-hover-color lg:px-7 ${
-                  activeTab === "/" ? "bg-primary text-white" : ""
+                className={`rounded-lg px-3 py-1 lg:px-7 transition-opacity duration-300 ${
+                  activeTab === "/"
+                    ? "bg-primary text-white"
+                    : "hover:opacity-75"
                 }`}
               >
                 Home
               </Link>
               <Link
                 href="/datasets"
-                className={`rounded-lg border-[1.5px] border-surface px-3 py-1 transition-colors duration-300 hover:border-hover-color lg:px-7 ${
-                  activeTab.includes("datasets") ? "bg-primary text-white" : ""
+                className={`rounded-lg px-3 py-1 lg:px-7 transition-opacity duration-300 ${
+                  activeTab.includes("datasets")
+                    ? "bg-primary text-white"
+                    : "hover:opacity-75"
                 }`}
               >
                 Datasets
               </Link>
               <Link
                 href="/themes"
-                className={`rounded-lg border-[1.5px] border-surface px-3 py-1 transition-colors duration-300 hover:border-hover-color lg:px-7 ${
-                  activeTab === "/themes" ? "bg-primary text-white" : ""
+                className={`rounded-lg px-3 py-1 lg:px-7 transition-opacity duration-300 ${
+                  activeTab === "/themes"
+                    ? "bg-primary text-white"
+                    : "hover:opacity-75"
                 }`}
               >
                 Themes
               </Link>
               <Link
                 href="/organizations"
-                className={`rounded-lg border-[1.5px] border-surface px-3 py-1 transition-colors duration-300 hover:border-hover-color lg:px-7 ${
-                  activeTab === "/organizations" ? "bg-primary text-white" : ""
+                className={`rounded-lg px-3 py-1 lg:px-7 transition-opacity duration-300 ${
+                  activeTab === "/organizations"
+                    ? "bg-primary text-white"
+                    : "hover:opacity-75"
                 }`}
               >
-                Organizations
+                {contentConfig.organizationName}
```

Comment:
As agreed, please put this back to `Organizations` and we create another story to rename everything to Publisher.